src/server_manager/model/server.ts

```diff
@@ -120,7 +120,7 @@ export interface ManagedServerRepository {
   getRegionMap(): Promise<Readonly<RegionMap>>;
   // Creates a server and returning it when it becomes active (i.e. the server has
   // created, not necessarily once shadowbox installation has finished).
-  createServer(region: RegionId): Promise<ManagedServer>;
+  createServer(region: RegionId, name: string): Promise<ManagedServer>;
```

Comment:
I don't see this function called in your PR. Shouldn't you change the call site?

Reply:
It's [here](https://github.com/Jigsaw-Code/outline-server/pull/446/files#diff-e3238a12d6d4f2299200994f4565a21fR726). (The file diff was collapsed).